src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -112,6 +112,7 @@ public function panelAction(Request $request, $token)
             'request' => $request,
             'templates' => $this->getTemplateManager()->getTemplates($profile),
             'is_ajax' => $request->isXmlHttpRequest(),
+            'all_query' => $request->query->all(),
```

Comment:
the Request is available in the template, so passing a new variable is not needed